api/pom.xml

```diff
@@ -19,21 +19,21 @@
 			<artifactId>fixture-factory</artifactId>
 			<version>3.1.0</version>
 		</dependency>
-		
+
 		<dependency>
```

Comment:
I have always been declaring all the dependencies in the main pom.xml and just reference them in different places including api and omod projects by just writing 2 lines of groupId and ArtifactId tags, other than repeating them in each project. Things that are declraed in the main pom are regarded as global and their references can be accessed anywhere else in the project including the config.xml and moduleApplicationContext.xml. Though, this NOT  a show stopper, but is the BEST practice for openMRS modules.